addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -38,19 +37,20 @@ class AccountMove(models.Model):
         ], string='Kode Transaksi', help='Dua digit pertama nomor pajak',
         readonly=False, copy=False,
         compute="_compute_kode_transaksi", store=True)
+    l10n_id_efaktur_range = fields.Many2one("l10n_id_efaktur.efaktur.range", string="E-faktur Range", copy=False)
```

Comment:
I would suggest adding the domain here and not in the view.
Also the domain should contain the company check